bitnami/common/templates/_labels.tpl

```diff
@@ -26,6 +27,7 @@ app.kubernetes.io/name: {{ include "common.names.name" . }}
 helm.sh/chart: {{ include "common.names.chart" . }}
 app.kubernetes.io/instance: {{ .Release.Name }}
 app.kubernetes.io/managed-by: {{ .Release.Service }}
+app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
```

Comment:
There's a main concern regarding this label to be included in the `common.labels.standard`: multi-application charts.

I mean, this works well in charts such as NGINX where there's a single component (`nginx`), see:

- https://github.com/bitnami/charts/tree/main/bitnami/nginx/templates

In these cases, it makes sense to set this label using `.Chart.AppVersion` since all the manifests refer to NGINX.

However, in charts such as Airflow where there are several components (airflow, airflow-scheduler, airflow-worker, etc) each of them having a different version, see:

- https://github.com/bitnami/charts/tree/main/bitnami/airflow/templates
- https://github.com/bitnami/charts/blob/main/bitnami/airflow/values.yaml

It makes less sense to use this approach, since we'll have components manifests (e.g. airflow-scheduler) with `app.kubernetes.io/version` label set with the value of the main component, which might be different.

What do you think @javsalgar @carrodher ?

Reply:
A possible approach would be to use "customLabels" argument received by `common.labels.standard` to overwrite the value of `app.kubernetes.io/version` on sub-components labels.

Reply:
Good idea! Please check this PR @javsalgar : https://github.com/bitnami/charts/pull/19223